backend/infrahub/lock.py

```diff
@@ -102,6 +102,9 @@ async def do_acquire(self, token: str) -> bool:
     async def release(self) -> None:
         await self.service.cache.delete(key=self.name)
 
+    async def locked(self) -> bool:
+        return await self.service.cache.get(key=self.name) is not None
+
```

Comment:
@fatih-acar I think I need this method on the `NATSLock` and I am not sure that this is correct

Reply:
LGTM :+1: